src/main/java/rx/Observable.java

```diff
@@ -350,8 +348,8 @@ protected Observable(OnSubscribe<T> f) {
      * @param <R> the resulting object type
      * @param converter the function that receives the current Observable instance and returns a value
      * @return the value returned by the function
+     * @since 1.3
```

Comment:
There's a lot of missing history below as well. Not sure if you were wanting it for every operator or not.